sqlmesh/core/model/definition.py

```diff
@@ -167,6 +168,8 @@ def render_definition(self, include_python: bool = True) -> t.List[exp.Expressio
         expressions = []
         comment = None
         for field_name, field_info in ModelMeta.all_field_infos().items():
+            if field_name == "fqn_":
```

Comment:
what's this, if this is a hidden property, maybe do _fqn

Reply:
It is effectively a derived field and therefore I need it to be part of the model (therefore can't be prefixed with `_`). I just don't want it to be included when we render a model since users should never see it.